packages/block-library/src/navigation/editor.scss

```diff
@@ -76,6 +77,13 @@
 	flex-wrap: wrap;
 }
 
+.wp-block-navigation {
+	min-height: 60px;
+	padding-top: 7px;
+	padding-bottom: 7px;
+	align-items: center;
```

Comment:
This is potentially a major change wrapped up here. I know Matias had a PR open to handle vertical alignment. I wonder whether it would be better to resolve this in that PR?

Reply:
it could be. Maybe we should remove these changes from here and delegate the vertical alignment to the other PR.

Reply:
However, I suggest keeping these changes until the new PR ships. Although it doesn't seem to be a solid solution, at least it gets visually better.